packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -224,6 +257,9 @@ export default class ClickableBehavior extends React.Component<Props, State> {
         super(props);
 
         this.state = startState;
+        this.waitingForClick = false;
+        this.enterClick = false;
+        this.triggers = getAppropriateTriggersForRole(this.props.role);
```

Comment:
I can't see people changing the `role` of a component ever, but we should handle updating all props properly.  In this case we should probably just call `getAppropriateTriggersForRole` whenever we need `triggers` (it's not very expensive) instead of storing it in an instance variable.